backend/lib/config/gardener.js

```diff
@@ -205,6 +210,10 @@ module.exports = {
       assert.ok(_.get(config, path), `Configuration value '${path}' is required`)
     })
 
+    _.set(config, 'sessionSecrets', [
+      config.sessionSecret,
+      ...(config.sessionSecretPrevious ? [config.sessionSecretPrevious] : [])
+    ])
```

Comment:
```suggestion
    const sessionSecrets = [config.sessionSecret]
    if (config.sessionSecretPrevious) {
      sessionSecrets.push(config.sessionSecretPrevious)
    }
    _.set(config, 'sessionSecrets', sessionSecrets)
```

Reply:
yeah I thought that it might make sense to have this code better readable, I will change it.. ✅